### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Shea690901**
A few minor corrections and please take a look into the todo...

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  search complete source for other occurances
+
+# Have Time #
+## documentation ##
+- complete docs/efun/contrib [packages/contrib]
+- check efun::shadowp/efun::query_shadowing
+- expand docs for efun:implode (missing three arg form)
+- missing docs for efun::defer
+
+## packages/contrib/ckntrib.cc ##
```
Comment (thefallentree) [inline TODO.md]: contrib.cc ?

Comment (Shea690901) [inline TODO.md]: Oops...
Yep, small typo ;)

Comment (thefallentree) [inline TODO.md]: well ?

**Comment by Shea690901** [comment]
Currently I've got slme problems with my network :(

**Comment by thefallentree** [comment]
sorry for the delay

**Pull request merged**